src/test/resources/test-rom/spec/programs/monitor_spec.lua

```diff
@@ -0,0 +1,11 @@
+local capture = require "test_helpers".capture_program
+
+describe("The monitor program", function()
+
+    it("displays its usage when given no argument", function()
+        multishell = nil
```

Comment:
Just curious as to the reason for this?

Reply:
Just a copy and paste mistake. I will remove it.